src/app/modules/time-clock/store/entry.effects.ts

```diff
@@ -219,6 +220,30 @@ export class EntryEffects {
     )
   );
 
+  @Effect()
+  updateLastEntryAndNew$: Observable<Action> = this.actions$.pipe(
+    ofType(actions.EntryActionTypes.UPDATE_TWO_ENTRIES),
+    map((action: actions.UpdateTwoEntries) => action.payload),
+    switchMap((entry) =>
+      this.entryService.loadEntries(new Date().getMonth() + 1).pipe(
+        map((entries) => {
+          const lastEntry = entries[1];
+          const isInLastEntry = moment(entry.start_date).isBefore(lastEntry.end_date);
```

Comment:
"isInLastEntry" does not make too much sense. Maybe we could change it for something like "isStartTimeInLastEntry" or something more descriptive.

Don't hesitate to use long names for variables or methods, just be wise to describe what's the purpose of the method/variable.

Reply:
I will take that into account the next time I add a name to my variables thanks, but this time I have chosen `isStartTimeInLastEntry`